ldm/invoke/generator/diffusers_pipeline.py

```diff
@@ -400,8 +400,15 @@ def device(self) -> torch.device:
     @property
     def _submodels(self) -> Sequence[torch.nn.Module]:
         module_names, _, _ = self.extract_init_dict(dict(self.config))
-        values = [getattr(self, name) for name in module_names.keys()]
-        return [m for m in values if isinstance(m, torch.nn.Module)]
+        submodels = []
+        for name in module_names.keys():
+            if hasattr(self, name):
```

Comment:
Is this necessary? Under diffusers 0.15 it seems like it is always self.config, and the other changes made in this PR would seem to make the code incompatible with 0.14.

Reply:
Ok, I tested with and without the conditional for self.config and it failed without the test. So I withdraw this comment.

Reply:
Hmm, ideally this shouldn't be necessary anymore. I'll try to open a PR to fix it